apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,292 @@
+package gov.cms.bfd.server.war.commons;
+
+/** Open API content short and value descriptions for each BFD API endpoint. */
+public final class OpenAPIContentProvider {
+  // NOTE: As much as we'd like to use Java 17 features for string concatentation,
+  // our code formatter completely mangles MUCH of the following concatenated text
+  // strings. So we'll fal back on tried-and-true structure to avoid
+  // unpleasantness.
+
+  /** Open API content short description for /Patient's identifier parameter. */
+  public static final String PATIENT_SP_RES_ID_SHORT =
+      "The patient resource identifier (_id) to search for";
+  /** Open API content value for /Patient's identifier parameter. */
+  public static final String PATIENT_SP_RES_ID_VALUE =
+      """
+    Fetch _Patient_ data using a FHIR _IdType_ identifier; an IdType
+    represents the logical identity for a resource, or as much of that
+    identity that is known. In FHIR, every resource must have a _logical ID_ which is
+    defined by the FHIR specification as:
+
+    `<code>Any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'),
```

Comment:
Let's tamp down on the usage of mixed markdown/html syntax, e.g. usage of backticks exclusively and drop the html tags for `<code>` and `</code>` _unless_ you find a use case for documenting a `<code>` tag (which is probably unlikely).